Application/EdFi.Ods.WebService.Tests/Controllers/VersionControllerTests.cs

```diff
@@ -27,7 +28,16 @@ public async Task VersionEndpointGetShouldBeValid()
             var json = await response.Content.ReadAsStringAsync();
 
             json.ShouldNotBeNullOrWhiteSpace();
-            Approvals.Verify(json);
```

Comment:
We'd have to update the version here when we update the constant but that's a good indication it's still working. Everything else seems like it would be pretty stable. should we keep the approval test?

Reply:
same reason as above, i changed it to deserializing the object and checking for the keys